NGitLab/IRepositoryClient.cs

```diff
@@ -8,6 +8,7 @@ namespace NGitLab
     public interface IRepositoryClient
     {
         ITagClient Tags { get; }
+        IReleaseClient Releases { get; }
```

Comment:
Can you add the new property at the end of the interface?

Reply:
I put it at the first best place where I saw another `Client`, I've moved it down now.